sound/soc/intel/boards/skl_hda_dsp_generic.c

```diff
@@ -150,6 +152,28 @@ static int skl_hda_fill_card_info(struct snd_soc_card *card,
 		}
 	}
 
+	if (!ctx->bt_offload_present) {
+		/* remove last link since bt audio offload is not supported */
+		num_links--;
```

Comment:
You define `#define BT_DAI_COUNT		1`, INHO, it is better to use `num_links -= BT_DAI_COUNT` in case BT_DAI_COUNT is changed one day.

Reply:
fixed